### PR TITLE
WAZO-2943: Wazo teams integration

### PR DESCRIPTION
**Pull request by ja0x**
Depends-On: https://github.com/wazo-platform/wazo-test-helpers/pull/81

Adds a plugin for chatd that allows Microsoft Teams to synchronize it's presence with Wazo.

Usage:
1. plugin is disabled by default, enable in config
    ``` yaml
    enabled_plugins:
      teams_presence: true
    ```
2. connect a user and enable `microsoft office365` integration
3. changes of presence in teams should be reflected in chatd
4. disable `microsoft office365` integration to stop presence synchronization

### CODE REVIEW

**Comment by ja0x** [comment]
recheck

**Comment by ja0x** [comment]
recheck

**Comment by ja0x** [comment]
recheck

**Review by pc-m — CHANGES_REQUESTED** [review]

File: wazo_chatd/plugins/teams/plugin.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2022-2022 The Wazo Authors  (see the AUTHORS file)
```
[the remark this answers is not in the document]
Comment (ja0x) [reply wazo_chatd/plugins/teams/plugin.py]: done

**Review by pc-m — APPROVED** [review]

**Pull request merged**